docs/terminology.md

```diff
@@ -37,7 +37,7 @@ See [Easyconfig files][easyconfig_files].
 
 <!-- XXX - UPDATE BY VERSION FIXME -->
 
-In EasyBuild version |version| the framework source code consists of about 19000 lines of code,
+In EasyBuild version {{ easybuild.version }} the framework source code consists of about 19000 lines of code,
```

Comment:
The counts (`19000 lines of code`) are also out of date.

So maybe we should go with a statement like:

```
In EasyBuild version 4.7.1 (released March 2023), the framework source code consists of about FIXME lines of code, ...
```

Reply:
Counts obtained using [cloc](https://github.com/AlDanial/cloc) for easybuild-framework v4.7.1:

```
$ cloc easybuild
     251 text files.
     250 unique files.
     251 files ignored.

github.com/AlDanial/cloc v 1.97  T=0.32 s (791.4 files/s, 192258.9 lines/s)
-------------------------------------------------------------------------------
Language                     files          blank        comment           code
-------------------------------------------------------------------------------
Python                         245          10066          18689          31491
Bourne Shell                     5            103             53            332
-------------------------------------------------------------------------------
SUM:                           250          10169          18742          31823
-------------------------------------------------------------------------------
```

There are 20 `__init__.py` files under `easybuild/`, those should not be counted as Python modules.

So there is ~31,500 lines of code in 225 Python modules, organized in about 20 Python package directories.